src/styled/table.ts

```diff
@@ -236,14 +237,29 @@ class Table<T extends object> {
     }
     shouldShorten()
 
+    // print table title
+    if (options.title) {
+      options.printLine(options.title)
+      // print title divider
+      options.printLine(''.padEnd(columns.reduce((sum, col) => sum + col.width!, 1), '='))
+    }
+
     // print headers
     if (!options['no-header']) {
-      let headers = ''
+      let headers = '| '
       for (const col of columns) {
         const header = col.header!
         headers += header.padEnd(col.width!)
       }
       options.printLine(chalk.bold(headers))
+
+      // print header dividers
+      let dividers = '| '
```

Comment:
Should the `|` devider also be between all columns? SHould both of these be configurable?

Reply:
@amphro Geordi brought that up during one of the previous design sessions and we basically all said in the name of saving space they weren't needed between columns. That's why they don't show up in the [env command spec](https://salesforce.quip.com/o26ZAMrvIbQZ#MIbACADJWRl).

I could make them configurable but I don't think it's necessary since it's a straight up improvement over the old format that was harder to read.